apis/sgv2-restapi/src/main/resources/application.yaml

```diff
@@ -47,10 +56,13 @@ quarkus:
     # force no grpc server start in dev as we have no grpc service
     dev-mode:
       force-server-start: false
+
+  # HTTP settings
   http:
     # 14-Sep-2022, tatu: Use the legacy port (same as SGv1) for now,
```

Comment:
Can just remove this comment now that we decided to use different ports.

Reply:
Done.